home/templates/home/home_page.html

```diff
@@ -1,11 +1,41 @@
 {% extends "base.html" %}
 {% load static %}
 {% load wagtailcore_tags %}
-
+ 
 {% block content %}
 
 <h1>A special template for the front page</h1>
 <p>
+<div class="row">
+  <div class="col-sm-5">
+    <div class="card">
+      <div class="card-body">
+        <h5 class="card-title">Special title treatment</h5>
+        <p class="card-text">With supporting text below as a natural lead-in to additional content.</p>
+        <a href="#" class="btn btn-primary">Go somewhere</a>
+      </div>
+    </div>
+  </div>
+  <div class="col-sm-5">
+    <div class="card">
+      <div class="card-body">
+        <h5 class="card-title">Special title treatment</h5>
+        <p class="card-text">With supporting text below as a natural lead-in to additional content.</p>
+        <a href="#" class="btn btn-primary">Go somewhere</a>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="col-sm-5">
+    <div class="card">
+      <div class="card-body">
+        <h5 class="card-title">Special title treatment</h5>
+        <p class="card-text">With supporting text below as a natural lead-in to additional content.</p>
+        <a href="#" class="btn btn-primary">Go somewhere</a>
+      </div>
+    </div>
+  </div>
+</div>
```

Comment:
This closing `</div>` doesn't match an opening, you should move the `col-sm-5` into the row above and change its width.

Reply:
This is done

Reply:
Are you sure that you have pushed the changes?

Reply:
The indentation of these HTML elements makes it really hard to read and see if the closing `</div>`s matches openings.